daemon/cmd/daemon_main.go

```diff
@@ -1291,6 +1285,12 @@ func runDaemon() {
 		d.SendNotification(monitorAPI.AgentNotifyStart, repr)
 	}
 
+	bootstrapStats.healthCheck.Start()
+	if option.Config.EnableHealthChecking {
+		d.initHealth()
+	}
+	bootstrapStats.healthCheck.End(true)
```

Comment:
Is there a particular reason for moving this?

Reply:
I think I preemptively moved it after our discussion, but didn't actually verify that it needed to be moved. My guess is no, but I'll double check.

Reply:
Not resolved?

Reply:
Removed. The initialization code didn't need to be moved.